src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 1; i <= 3; i++) {
```

Comment:
make 3 as `BALLS_TO_CREATE` constant
```suggestion
        for (int i = 0; i <= BALLS_TO_CREATE; i++) {
```